docs/api/samples/concept-scheme.json

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
+        "dc": "http://purl.org/dc/elements/1.1/",
+        "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+        "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+        "skos": "http://www.w3.org/2004/02/skos/core#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#"
+    },
+    "@id": "http://example.com/taxonomy",
```

Comment:
Scheme doesn't conform to variable represented in `management.oas3.yaml#/parameters/concept` may need another identifier for this, or for the domain?